templates/telemetry/firefox_kpi_dashboard_v1/query.sql

```diff
@@ -121,7 +163,27 @@ WITH
     'actual' AS type,
     submission_date AS date,
     id_bucket,
-    SUM(seen_in_tier1_country_mau) AS mau
+    -- We have a special way of determining whether to include a user in tier1
+    -- to match historical definition of tier1 FxA mau.
+    SUM(seen_in_tier1_country_mau) AS mau,
+    SUM(
+    IF
+      (country IN ('US',
+          'FR',
+          'DE',
+          'GB',
+          'CA'),
+        wau,
+        0)) AS wau,
```

Comment:
i suppose we don't have a historical definition of tier1 FxA wau that we need to match, and we're already mixing the definitions in this PR, but it seems odd to use both "seen in tier1" and "last seen in tier1" in the same row.

Reply:
Agreed it's a bit weird, but we generally have agreement to use the last seen values, and I think it's good to keep the "seen in tier 1" exception as small as possible.